### DISCUSSION TITLE
v0.2 - Fix problem when running on battery power

### DISCUSSION THREAD

**Discussion by c3d** [c3d/db48x#61] [Announcements]
The primary reason for a v0.2 is that this solves #12, a hang on battery power. There are also major bug fixes on integer arithmetic, and a new formatting of numbers.

**Full Changelog**: https://github.com/c3d/DB48X-on-DM42/compare/v0.1...v0.2

<hr /><em>This discussion was created from the release <a href='https://github.com/c3d/DB48X-on-DM42/releases/tag/v0.2'>v0.2 - Fix problem when running on battery power</a>.</em>

**Comment by balta1980** [comment]
I will check it ASAP
